unittests/AST/TensorFlow.cpp

```diff
@@ -99,3 +104,78 @@ TEST_F(TypeContainsTensorFlowValueTest, WorksForRecursiveTypes) {
   recursiveDecl->addMember(varDecl);
   EXPECT_FALSE(containsTensorFlowValue(recursiveType));
 }
+
+TEST_F(TypeContainsTensorFlowValueTest, TensorFlowType) {
+  EXPECT_TRUE(tf::isTensorHandle(tensorHandleType));
+  EXPECT_FALSE(tf::isTensorHandle(floatType));
+}
+
+TEST_F(TypeContainsTensorFlowValueTest, TensorHandleElementType) {
+  EXPECT_TRUE(tf::getTensorHandleElementType(floatType).isNull());
```

Comment:
Can you also add a test to check that the function extracts the correct type?

Reply:
sure. I am adding that check on my next commit.